ios/Sources/TimetableFeature/Detail/TimetableDetailFeature.swift

```diff
@@ -0,0 +1,36 @@
+import ComposableArchitecture
+import Foundation
+import Model
+
+public struct TimetableDetailState: Equatable {
+    public var timetable: AnyTimetableItem
+    public var isSheetPresented: URL?
+
+    var isShowingSheet: Bool {
+        isSheetPresented != nil
+    }
```

Comment:
How about this?  `isSheetPresented` causes misunderstanding.

```suggestion
    public var showingURL: URL?

    var isSheetPresented: Bool {
        showingURL != nil
    }
```